tests/ui/positional_named_format_parameters.rs

```diff
@@ -0,0 +1,56 @@
+// run-rustfix
+#![allow(unused_must_use)]
+#![allow(named_arguments_used_positionally)] // Unstable at time of writing.
```

Comment:
Is there any overlap between this lint and `named_arguments_used_positionally`? `clippy::positional_named_format_parameters` will reach stable at the same time or after `named_arguments_used_positionally` does so ideally we wouldn't be double warning

Reply:
Yes for a very specific case (when a named argument is used only once positionally, is never named and does not have a width, modifier or precision) so it should be fine.

Additionally inhibiting such a case may prove more confusing to the end user since they would expect this lint to catch such a case.